subsys/nrf_security/src/drivers/cracen/ext/Kconfig

```diff
@@ -7,7 +7,7 @@ if PSA_CRYPTO_DRIVER_CRACEN
 
 config CRACEN_LOAD_MICROCODE
 	bool
-	depends on SOC_SERIES_NRF54LX || SOC_NRF54H20_CPUSEC
+	depends on SOC_SERIES_NRF54LX || SOC_SERIES_NRF54HX
```

Comment:
I thought SD-ROM code loads the microcode but let's keep `SOC_SERIES_NRF54HX` here until we are sure for all PDK's